src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/InputJavadocComment.javadoc

```diff
@@ -1,4 +1,4 @@
 <p>
 My {@code Class}
 </p>
-@see never
\ No newline at end of file
+@see never
```

Comment:
Does impractical mean impossible? Because if EOF is possible there I think we should support it.